src/cs-core/InputManager.hpp

```diff
@@ -35,6 +35,10 @@ class ScreenSpaceGuiArea;
 
 namespace cs::core {
 
+#ifdef _MSC_VER
+#pragma warning(disable : 4275)
+#endif _MSC_VER
```

Comment:
Should we do something about this warning? It is currently the result of subclassing ViSTA classes that don't declare a DLL interface.

Reply:
If it just requires adding a missing `VISTAKERNELAPI` to the `IVistaDirectKeySink`, we should definitely do this.